### PR TITLE
[FIX] crm_claim_filter: fixed name_search behaviour

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

**Comment by oihane** [comment]
@Daniel-CA @mikelarre @agaldona review please

File: crm_claim_filter/models/res_partner.py [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+from openerp import models
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
-    @api.model
-    def name_search(self, name='', args=None, operator='ilike', limit=100):
+    def name_search(self, cr, uid, name, args=None, operator='ilike',
+                    context=None, limit=100):
```
Comment (oihane) [inline crm_claim_filter/models/res_partner.py]: https://github.com/OCA/OCB/blob/8.0/openerp/addons/base/res/res_partner.py#L659

**Comment by Daniel-CA** [comment]
:+1:

**Comment by esthermm** [comment]
:+1:

**Comment by agaldona** [comment]
:+1:

**Pull request merged**